src/app/__init__.py

```diff
@@ -11,12 +11,43 @@
 
 app = Flask(__name__, static_url_path=cfg.WEBSITE["static-url-path"])
 
-if 'DEBUG' in environ:
+if 'DEBUG' in environ and environ['DEBUG'] == "True":
```

Comment:
Broke: 

`if 'DEBUG' in environ and environ['DEBUG'] == "True":`

Woke:

`environ.get("DEBUG") == "True":`